index.bs

```diff
@@ -85,8 +88,8 @@ The DeviceOrientationSensor Interface {#deviceorientationsensor-interface}
 <pre class="idl">
   [Constructor(optional SensorOptions sensorOptions)]
   interface DeviceOrientationSensor : Sensor {
-    void populateQuaternion(Float32Array array);
-    void populateMatrix(Float32Array array);
+    void populateQuaternion(Float32Array buffer);
```

Comment:
I think that buffer makes it more clear that we write to these... we could use targetBuffer or similar